Matrix.C

```diff
@@ -842,7 +842,7 @@ Matrix::qrcp_pivots_transpose_serial(int* row_pivot,
   double* work = new double[lwork];
   double* tau  = new double[std::min(num_rows_of_transpose,
 				     num_cols_of_transpose)];
-  int* pivot = new int[num_cols_of_transpose];
+  int* pivot = new int[num_cols_of_transpose] ();
```

Comment:
So before, this was randomly initialized. Depending on its initialization, dgeqp3 was returning a different answer. I initialized it to all zeroes and the test passes. I'm not sure if this is correct though.

http://www.netlib.org/lapack/explore-html/dd/d9a/group__double_g_ecomputational_ga1b0500f49e03d2771b797c6e88adabbb.html